RELEASING.md

```diff
@@ -1,11 +1,11 @@
-1. `bundle exec fastlane update_hybrid_common version:x.x.x` to set the purchases-hybrid-common version number.
-1. Update CHANGELOG.latest.md with the changes for the current version (to be used by Fastlane for the github release notes).
-1. `bundle exec fastlane bump_and_update_changelog version:x.x.x` to set the version number.
-1. Update versions in VERSIONS.md.
-1. Run `npm run build`
-1. Open a PR from branch `release/x.x.x` against `main`
-1. Merge to `main` and pull from your machine
-1. `git tag x.y.z`
-1. `git push origin --tags`
-1. Create a new release in github and upload
-1. `npm publish`
+### Releasing a version:
+
+1. Create a `fastlane/.env` file with your GitHub API token (see `fastlane/.env.SAMPLE`). This will be used to create the PR, so you should use your own token so the PR gets assigned to you.
+1. Run `bundle exec fastlane bump`
+    1. Input new version number
+    2. Update CHANGELOG.latest.md to include the latest changes. Call out API changes (if any). You can use the existing CHANGELOG.md as a base for formatting. To compile the changelog, you can compare the changes between the base branch for the release (usually main) against the latest release, by checking https://github.com/revenuecat/react-native-purchases/compare/<latest_release>...<base_branch>. For example, https://github.com/revenuecat/react-native-purchases/compare/3.10.0...main.
+    3. A new branch and PR will automatically be created
+1. `bundle exec fastlane update_hybrid_common version:x.x.x` to set the purchases-hybrid-common version number if needed.
+1. Run `npm run build` and make sure it works. If any issues appear, fix them first.
+1. Wait until PR approved and make sure local copy matches remote
```

Comment:
Maybe I would be more specific and say pull from repository. I was a bit confused about the "make sure local copy matches remote"